### ISSUE TITLE
upgrade to agave v2 from solana v1.18 packages

### ISSUE THREAD

**Issue by EmilRejman** [helius-labs/helius-rust-sdk#91]
Hi!

Most of the valdiators already use agave-v2.0+ or dependent builds.

In your dependencies has each solana crate labled with `^1.18.12` which prevent builds.

Example:

```
error: failed to select a version for the requirement `solana_rbpf = "=0.8.0"`
candidate versions found which didn't match: 0.8.5, 0.8.4, 0.8.3, ...
location searched: crates.io index
required by package `solana-program-runtime v1.18.12`
    ... which satisfies dependency `solana-program-runtime = "=1.18.12"` of package `solana-config-program v1.18.12`
    ... which satisfies dependency `solana-config-program = "=1.18.12"` of package `solana-account-decoder v1.18.12`
    ... which satisfies dependency `solana-account-decoder = "^1.18.12"` of package `helius v0.2.1`
    ... which satisfies dependency `helius = "^0.2.1"` of package (...)
```

**Comment by 0xIchigo** [comment]
gm! I will be cutting a new release for the SDK later today. This issue will be resolved then, as #80 covers this version bump

**Comment by 0xIchigo** [comment]
Can be considered closed now with v0.2.2 being published
